common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -179,12 +179,16 @@ div.problem {
 
   form > label, .problem-group-label {
     display: block;
-    margin-bottom: 0;
+    margin-bottom: $baseline;
     font: inherit;
     color: inherit;
     -webkit-font-smoothing: initial;
   }
 
+  .problem-group-label + .question-description {
+    margin-top: -20px;
```

Comment:
Yes. Not the clean way to achieve this. The HTML could have been structured in a different way, giving the description can be absent, the margin could be applied to a div, rather than p tag or label tag.